src/blueprints/tv2/action-factories/tv2-replay-action-factory.ts

```diff
@@ -69,18 +69,18 @@ export class Tv2ReplayActionFactory {
     }
   }
 
-  private removeAllWhitespace(value: string): string {
-    return value.replaceAll(' ', '')
+  private getSanitizedName(source: Tv2SourceMappingWithSound): string {
+    return source.name.replaceAll(' ', '').replaceAll('/', '_')
   }
 
```

Comment:
I don't know how useful this logic is, but in order not to repeat it somewhere, we can export what to replace and with what in the parameters of the function.

```
 private stringReplace(value: string, valueForChange: string, changeTo: string): string {
    return value.replaceAll(valueForChange, changeTo)
  }
```
`valueForChange: string, changeTo: string` can be string[] if we need to cover change multiple parts of the string.

SIDE NODE: that way the function witch call `stringReplace` will know what exactly what will be change it, without following the function.